styles/stylesheet.css

```diff
@@ -0,0 +1,275 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: white;
+	height: 100%;
+	padding: 1rem;
+}
+
+.header {
+	background-repeat: no-repeat;
+	max-width: 100%;
+	height: 25rem;
+	background-image: url("../images/header-image.jpg");
+	background-size: cover;
+}
+
+.header-logo {
+	max-width: 100%;
+	height: auto;
+	float: left;
+	padding: 2rem 2rem 1rem 2rem;
+}
+
+.header-logo img {
+	float: left;
+}
+
+.slogan {
+	font-family: "Comic Sans MS", cursive, sans-serif;
+	font-size: 1.5rem;
+	color: #1B8F1B
+}
+
+.shoppingcart {
+	float: right;
+	padding: 1rem;
+}
+
+.shoppingcart img {
+	width: 50px;
+	height: auto;
+}
+
+.shoppingcart p {
+	font-family: 'Arvo', serif;
+	font-size: 1.5rem;
+	color: white;
+}
+
+.checkout-button {
+	background-color: #4CAF50;
+	border: none;
+	color: white;
+	font-family: 'Arvo', serif;
+	padding: 15px 32px;
+	font-size: 1rem;
+	cursor: pointer;
+}
+
+nav {
+	overflow: hidden;
+	background-color: #333;
+	text-align: center;
+}
+
+nav ul {
+	margin: 0;
+	list-style-type: none;
+	width: 100%;
+	padding: 2rem;
+}
+
+nav li,
+nav li a {
+	display: inline;
+	color: #f2f2f2;
+	padding: 14px 16px;
+	text-decoration: none;
+	font-size: 2rem;
+	clear: both;
+}
+
+nav li a:hover,
+nav li a:active {
+	background-color: #ddd;
+	color: black;
+}
+
+.flex-container {
+	display: flex;
+	flex-direction: column;
+	align-items: stretch;
+}
+
+.flex-container::after {
+	content: "";
+	clear: both;
+	display: table;
+}
+
+.box {
+	flex: 2;
+	background-color: white;
+	padding-left: 3rem;
+}
+
+.box img {
+	float: left;
+	padding-top: 3rem;
+	max-width: 100%;
+	height: auto;
+}
+
+.container-about {
+	font-family: 'Arvo', serif;
+	font-size: 2rem;
+	display: flex;
+	flex-wrap: wrap;
+	padding: 0.5rem;
+}
+
+.container-about h3 {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	font-size: 3rem;
+	font-weight: bold;
+	position: relative;
+}
+
+.container-about p {
+	color: black;
+	font-family: 'Arvo', serif;
+	font-size: 1rem;
+	position: relative;
+}
+
+.blog-post h3 {
+	padding: 0.5rem;
+}
+
+.blog-post p {
+	padding: 0.5rem;
+	font-size: 1rem;
+}
+
+.blog-post img {
+	padding: 0.5rem;
+}
+
+.grid-container {
+	display: grid;
+	grid-template-columns: auto auto auto auto;
+	grid-column-gap: 2rem;
+	padding: 10px;
+	max-width: 1200px;
```

Comment:
This is causing your grid not to be responsive. Usually you will never want to define a exact width, because every screen has a different width. Try removing max-width entirely and see if this helps your grid on mobile.